ApiReview/API.Polly.Core/NoDocs/Polly.Strategy/IResilienceArguments.cs

```diff
@@ -0,0 +1,8 @@
+// Assembly 'Polly.Core'
+
+namespace Polly.Strategy;
+
+public interface IResilienceArguments
```

Comment:
I don't understand the value of this interface. Nobody will ever program against an IResilienceArgument, so what's the point?